sdks/go/pkg/beam/io/parquetio/parquetio.go

```diff
@@ -118,6 +122,19 @@ func (a *parquetReadFn) ProcessElement(ctx context.Context, filename string, emi
 	return nil
 }
 
+// Write writes a PCollection<parquetStruct> to .parquet file.
+// Write expects a type t of struct with parquet tags
+// For example:
+// type Student struct {
+//   Name    string  `parquet:"name=name, type=BYTE_ARRAY, convertedtype=UTF8, encoding=PLAIN_DICTIONARY"`
+//   Age     int32   `parquet:"name=age, type=INT32, encoding=PLAIN"`
+//   Id      int64   `parquet:"name=id, type=INT64"`
+//   Weight  float32 `parquet:"name=weight, type=FLOAT"`
+//   Sex     bool    `parquet:"name=sex, type=BOOLEAN"`
+//   Day     int32   `parquet:"name=day, type=INT32, convertedtype=DATE"`
+//   Ignored int32   //without parquet tag and won't write
+// }
+
```

Comment:
```suggestion
// }
```

Reply:
Ah, the blank line prevents associating the comment with the function. 

The PR doesn't have "allow committers to make changes" set otherwise I'd commit the change and then merge the PR.